### PR TITLE
Query logging enabled with ENV variable

### PR DESCRIPTION
**Pull request by trepik**
First try to solve #13 
(not tested)

### CODE REVIEW

File: 5.5/README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ The following environment variables influence the MySQL configuration file. They
 |  `MYSQL_INNODB_LOG_BUFFER_SIZE` | The size of the buffer that InnoDB uses to write to the log files on disk | 8M (or 15% of available memory)
 |  `MYSQL_DEFAULTS_FILE`          | Point to an alternative configuration file                        |  /etc/my.cnf
 |  `MYSQL_BINLOG_FORMAT`          | Set sets the binlog format, supported values are `row` and `statement` | statement
+|  `MYSQL_LOG_QUERIES`            | Turns on logging of queries | 0
```
Comment (bparees) [inline 5.5/README.md]: we've generally tried to use true/false (and name it MYSQL_LOG_QUERIES_ENABLED), is "0" and "1" the values mysql requires for its actual configuration to enable this?

**Comment by bparees** [comment]
[test]

File: 5.5/README.md (hunk 1 above)
Comment (trepik) [inline 5.5/README.md]: "By default, the general query log is disabled. To specify the initial general query log state explicitly, use --general_log[={0|1}]. With no argument or an argument of 1, --general_log enables the log. With an argument of 0, this option disables the log. To specify a log file name, use --general_log_file=file_name."
from [DOC](http://dev.mysql.com/doc/refman/5.7/en/query-log.html)

Comment (bparees) [inline 5.5/README.md]: ok thanks,  can you add "set to 1 to enable" or something to the description?

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
lgtm.  @hhorak @praiskup any comments?

**Comment by bparees** [comment]
oh and @trepik please squash your commits.

**Comment by bparees** [comment]
(@hhorak you might want to turn on PR merge squashing for all the container repos: https://help.github.com/articles/configuring-pull-request-merge-squashing/)

**Comment by hhorak** [comment]
Looks good to me, LGTM. The squashing is already enabled, but I'm not sure whether it works the way I think it does :) Let's try it..

**Comment by hhorak** [comment]
It looks the squashing works fine, thanks for pointing to that, @bparees.

**Comment by bparees** [comment]
> Looks good to me, LGTM. The squashing is already enabled

weird, I only had a "merge" button and not a "squash+merge" which i was expecting to see.

**Comment by hhorak** [comment]
> weird, I only had a "merge" button and not a "squash+merge" which i was expecting to see.

I was looking for it few seconds as well, there is a possibility to change an action on the page with commit message.

**Comment by bparees** [comment]
ah, thanks!

**Pull request merged**